### ISSUE TITLE
Liquidity pool tool doesn't like my username?

### ISSUE THREAD

**Issue by grctest** [bitshares/bitshares-ui#3616]
**Describe the bug**

I can't perform a search for the UIA to be converted to a liquidity pool token with my account 'nft.artist'.

I therefore can't create a liquidity pool with this account.

**To Reproduce**

Explore to the account `nft.artist`, then go to advanced -> pools -> Create.

Within the first field, search for NFTEA.

Nothing happens, however the console reports the following errors:

```
getAssetsByIssuer id =  nft.artist_10
{
    "code": 7,
    "message": "Execution error: Bad Cast: Invalid cast from type 'null_type' to Object",
    "data": {
        "code": 7,
        "name": "bad_cast_exception",
        "message": "Bad Cast",
        "stack": [
            {
                "context": {
                    "level": "error",
                    "file": "variant.cpp",
                    "line": 579,
                    "method": "get_string",
                    "hostname": "",
                    "thread_name": "th_a",
                    "timestamp": "2023-01-13T00:48:24"
                },
                "format": "Invalid cast from type '${type}' to Object",
                "data": {
                    "type": "null_type"
                }
            },
            {
                "context": {
                    "level": "warn",
                    "file": "object_id.hpp",
                    "line": 265,
                    "method": "from_variant",
                    "hostname": "",
                    "thread_name": "th_a",
                    "timestamp": "2023-01-13T00:48:24"
                },
                "format": "",
                "data": {
                    "var": null
                }
            }
        ]
    }
}
```

If I use another account which doesn't include a dot in the middle, the first field's search box works as expected.

**Expected behavior**
I'm able to create a liquidity pool token with this account, getting past the first field.

**Desktop (please complete the following information):**
Desktop, windows 11.
Occurs on web too.

**Comment by abitmore** [comment]
It looks like the account `nft.artist` doesn't own any asset?

**Comment by grctest** [comment]
For an account with no issued assets and no dot in name, I get the following warnings when attempting to search in the first field:
```
onGetLiquidityPoolsAccount
getAssetsByIssuer id =  abcde12345_10
CreatePooModal marketsList:  Array(0)
```

But for nft.artist I get:
```
getAssetsByIssuer id =  nft.artist_10
Error in AssetActions.getAssetList:  Object
```

Similar for in.abit:
```
getAssetsByIssuer id =  in.abit_10
Error in AssetActions.getAssetList:  Object
```
Accounts without a dot in their username seem to work as expected, but those with a dot don't work?

**Comment by abitmore** [comment]
Another difference is that the account `abcde12345` owns one asset `"id":"1.3.5544","symbol":"NORNS"`, but the account `nft.artist` doesn't own one.

I tried another account `abcde1234`, then got the same error `Error in AssetActions.getAssetList:  Object`.

The RPC call sent to the API node is:
```
{"method":"call","params":[0,"get_assets_by_issuer",["abcde1234",null,10]],"id":1161}
```

It's the `null` in the call that caused the error.

Actually, according to the code in `onPoolNameChange(getBackedAssets, e)`: https://github.com/bitshares/bitshares-ui/blob/ce9373970a27692468627915b334c74886aa0efd/app/components/Modal/CreatePoolModal.jsx#L337-L350

on line 338, `assets` is assigned an empty array when the account doesn't own any asset, then `asset[0]` aka `null` is passed into `lookupAccountAssets()` on line 345, then: https://github.com/bitshares/bitshares-ui/blob/ce9373970a27692468627915b334c74886aa0efd/app/components/Modal/MarketPickerHelpers.js#L18-L24

`null` is passed as `start` to `getAssetsByIssuer(issuer, count, start, includeGateways = false)`, then: https://github.com/bitshares/bitshares-ui/blob/9e4d1d823015c3c198d94aed458c7677f91e7890/app/actions/AssetActions.js#L588-L599

`null` is passed to the `get_assets_by_issuer` API call.

So it is actually a UX issue. Simply adding a size check for `assets` will hide the error, but won't improve UX (still nothing happens if the account doesn't own an asset). We can show more text on the "create pool" page, e.g. the prerequisites to create a pool "the account need to own an asset which is a simple asset and has 0 supply and not linked to another liquidity pool", error message "you don't own any asset" or "you don't own an asset that contains [the input]", etc.

**Comment by grctest** [comment]
Indeed, the uncaught errors and flawed UX is indeed poor quality programming.

**Comment by grctest** [comment]
> So it is actually a UX issue.

Idd, the current UX is poorly designed. It should be more descriptive and potentially ask for each in a wizard format rather than all up front. I had to ask others to realize I had to first create an UIA on an entirely different page for the liquidity pool asset to even exist, this was not clear at all from the UI.
